test/Tests/src/Data/Statistics_Spec.enso

```diff
@@ -41,121 +37,121 @@ spec =
             text_set.compute . should_equal 3
 
         Test.specify "should be able to get minimum of maximum values" <|
-            simple_set.compute Minimum . should_equal 1
-            number_set.compute Minimum . should_equal -45.84 epsilon=double_error
-            missing_set.compute Minimum . should_equal -41.02 epsilon=double_error
-            with_nans_set.compute Minimum . should_equal -41.02 epsilon=double_error
-            text_set.compute Minimum . should_equal "A"
-            simple_set.compute Maximum . should_equal 5
-            number_set.compute Maximum . should_equal 42.01 epsilon=double_error
-            missing_set.compute Maximum . should_equal 33.85 epsilon=double_error
-            with_nans_set.compute Maximum . should_equal 33.85 epsilon=double_error
-            text_set.compute Maximum . should_equal "D"
+            simple_set.compute Statistics.Minimum . should_equal 1
```

Comment:
This is just my personal and subjective opinion, but I don't like this change.

This adds a _lot_ of typing to the tests, and given that currently the parser does not support multi-line expressions (at least not in all contexts we'd need), the lines are already pretty long and adding these prefixes makes them even longer.

While I do understand we may not want to pollute the namespace too much in some places, I don't see why this would be a problem in tests, especially as the `Minimum` etc. statistics are the core of what is tested.